src/map/lua/lua_baseentity.cpp

```diff
@@ -10509,26 +10509,34 @@ inline int32 CLuaBaseEntity::dispelAllStatusEffect(lua_State *L)
 *  Notes   : Returns a Lua table with the information on the Status Effect stolen
```

Comment:
since it no longer returns a lua table, should probably update this

Reply:
Can do.